js/bundledPlugins/previouslyUsedKeywords.js

```diff
@@ -0,0 +1,86 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var isUndefined = require( "lodash/isUndefined" );
+
+/**
+ * Constructs the PreviouslyUsedKeywords plugin
+ * @param {object} app The app
+ * @param {object} usedKeywords An object with keywords and ids where they are used
+ * @param {string} searchUrl The url used to link to a search page when multiple usages of the keyword are found
+ * @param {string} postUrl The url used to link to a post when 1 usage of the keyword is found
+ * @constructor
+ */
+var PreviouslyUsedKeywords = function( app, usedKeywords, searchUrl, postUrl ) {
+	this.app = app;
+	this.usedKeywords = usedKeywords;
+	this.searchUrl = searchUrl;
+	this.postUrl = postUrl;
+	this.i18n = this.app.i18n;
```

Comment:
Why does this object need to know where the i18n object is located on the app. i18n should just be a parameter.